EIPS/eip-5269.md

```diff
@@ -0,0 +1,35 @@
+---
+eip: 5269
+title: Human Readable Interface Detection
+description: An interface returning Human Readable Interface
+author: Zainan Victor Zhou (xinbenlv@)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2022-07-15
+requires (*optional): <EIP number(s)>
```

Comment:
```suggestion
requires: 165
```

Reply:
Not requiring 165

Reply:
`requires` in this context means "this EIP references another EIP and to fully understand it you may need to read the referenced EIPs".  If you don't want to require 165, my recommendation would be to remove it from the abstract.  Generally speaking, I recommend authoring EIPs such that they stand alone without depending on the reader knowing about others EIPs that aren't hard requirements.  In this case, the abstract should just describe this EIP and simply not mention alternatives.